app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailPresenter.java

```diff
@@ -46,12 +48,27 @@ public void attach(IEventDetailView eventDetailView, long initialEventId) {
         this.initialEventId = initialEventId;
     }
 
+    private void loadAll(boolean refresh) {
```

Comment:
Thats fine. But I think `start()` method of presenter should carry a parameter which will notify presenter if it needs to load data from disk. Currently every time start is called presenter loads data again.

Also this will fix #196 
@iamareebjamal what do you say